README.md

```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
+- Install `click` Python package, `pip install click` (required for the google-oauthlib-tool)
 - Use the [`google-oauthlib-tool`][google-oauthlib-tool] to generate credentials:
 ```
-pip install google-auth-oauthlib[tool]
+pip install google-auth-oauthlib-tool
```

Comment:
Did the previous command, `pip install google-auth-oauthlib[tool]`, fail?

Reply:
Yes it failed

Reply:
Did you follow [the steps here](https://developers.google.com/assistant/sdk/develop/grpc/integrate#authorize_and_authenticate_your_google_account_to_work_with_the_assistant) to create a virtual Python environment before running the pip commands?

Reply:
@championswimmer make sure you updated pip and setuptools with:
```
pip install pip setuptools --upgrade
```